docs/Developers/Newcomers_Guide.rst

```diff
@@ -203,6 +205,9 @@ into your bash. This will analyze your code and help you fix it.
 Step 5. Sending Your Changes
```

Comment:
above at line 199 (https://github.com/coala/coala/pull/3657/commits/5fa9461c8eb686d2053a0f4b9829d6a706b055ba?diff=split#diff-604303f80032b9f21fe7a80f8deda96cR199) I'd add that you should always do this and also that this relates to gitmate errors, that will confuse people less